nautobot_circuit_maintenance/models.py

```diff
@@ -133,7 +134,7 @@ class Note(OrganizationalModel):
     """Model for maintenance notes."""
 
     maintenance = models.ForeignKey(CircuitMaintenance, on_delete=models.CASCADE, default=None)
-    title = models.TextField()
+    title = models.CharField(max_length=500)
```

Comment:
I thought that the issue with Django + MySQL + TextField was only an issue with fields that have `unique=True`? Why does `title` need to be changed from `TextField` to `CharField`, but `comment` (below) does not?

Reply:
It's actually the case: https://github.com/nautobot/nautobot-plugin-circuit-maintenance/blob/develop/nautobot_circuit_maintenance/models.py#L151

Reply:
I missed that, thanks!

What do you think about maybe limiting this to 100 or 200 characters? 500 seems excessive for a `title` string, unless you're aware of cases that already exist where we might need that many?

The limit should be another constant in any case, rather than a literal `500` here. Also we should make sure that anywhere in the code where we're creating or updating a Note's `title` has safety code to truncate the string to fit into the DB field if needed, similar to what we have for maintenance names, notification subject lines, etc.

Reply:
Ok, let's limit it to the 200 characters, and truncate it when necessary.